hr_cae_event_timesheet_sheet/tests/test_event.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Manuel Claeys Bouuaert <manuel@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import tests
+
+
+class TestEvent(tests.common.TransactionCase):
```

Comment:
I love reading tests! ;-) 

It's better practice to write more "unitary" tests, that is "one test per scenario". Here for example, it'd be easier to test / fix / read (and better documentation) if you had separate tests for `partner_1`, `partner_3` and `employee` (case in point, I don't understand what  `partner_2` adds to `partner_1`)

```python
def test_create_portal_user_from_registrated_partner(self):
    ...
def test_open_registration_doe_not_log_time_sheet(self):
    ...
def test_create_portal_user_from_registrated_employee(self):
    ...
```

Reply:
I was thinking about this when cycling back home. Will do so! Partner 2 doesn't have the partner_id linked in it's registration.